hunting/windows/docs/detect_dll_hijack_via_masquerading_as_microsoft_native_libraries_elastic_defend.md

```diff
@@ -0,0 +1,43 @@
+# Detect DLL Hijack via Masquerading as Microsoft Native Libraries - Elastic Defend
```

Comment:
```suggestion
# Detect DLL Hijack via Masquerading as Microsoft Native Libraries
```

It would seem we do not need to add the integration to the end of the name in the header. We already have the integration field in the metadata.

Reply:
There are many cases where the name, intent, description, and notes are the same, with the only differences being the target integration.  In this case, there is a separate hunt query targeting the windows integration + sysmon. The only way to differentiate the 2 in an identifiable way (beyond UUID) is this suffixed title descriptor

* Detect DLL Hijack via Masquerading as Microsoft Native Libraries - Elastic Defend
* Detect DLL Hijack via Masquerading as Microsoft Native Libraries - Sysmon

Reply:
ah okay, maybe we allow `query` to be an array of strings and reduce redundancy some.

Reply:
That was an option discussed, though it would need to be a dict of `integration: query` at the least. Pros and cons for both approaches